drivers/i3c/i3c_renesas_imx3112.c

```diff
@@ -0,0 +1,414 @@
+/* renesas_imx3112_i3c.c - I3C file for Renesas I3C 1:2 bus multiplexer */
+
+/*
+ * Copyright (c) 2022 Google LLC.
```

Comment:
2023?

Reply:
I made the initial commits in 2022, i'm just upstreaming it this year.

Reply:
I will resolve this if that's okay.